doc/source/admin/scaling.md

```diff
@@ -237,160 +154,121 @@ Handlers (as well as assignment methods) are not configurable when using **uWSGI
 
 ## Configuration
 
-### uWSGI
+### Gunicorn
 
-Although this document goes in to significant detail about uWSGI configuration, many more options are available, as well
-as additional documentation about options described here. Consult the uWSGI documentation for more:
+We will only outline a few of Gunicorn's options, consult the [Gunicorn documentation](https://docs.gunicorn.org/en/latest/settings.html) for more.
 
-* [Configuring uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html)
-* [uWSGI Options](https://uwsgi-docs.readthedocs.io/en/latest/Options.html)
-* [Quickstart for Python/WSGI applications](https://uwsgi-docs.readthedocs.io/en/latest/WSGIquickstart.html)
 
-Configuration is performed in the `uwsgi` section of `galaxy.yml`. You will find that the default, if copied from
-`galaxy.yml.sample`, is commented out. The default configuration options are provided to uWSGI on the command line by
-Galaxy's `run.sh` script.
+Note that by default Galaxy will use [gravity](https://github.com/galaxyproject/gravity/) to create a [supervisor](http://supervisord.org/) configuration that
+uses Gunicorn configuration values read from the `gravity` section of your `galaxy.yml` file.
+This is the preferred and out-of-the box way of configuring Gunicorn for serving Galaxy.
+If you are not using `./run.sh` for starting Galaxy or you would like to use another process manager
+all of the Gunicorn configuration values can also directly be set on the command line.
 
-Galaxy releases prior to 18.01 (or upgraded-to-18.01+ servers which have not migrated their configuration to the YAML
-format) used an INI-format configuration file, `galaxy.ini`.
+Configuration is performed in the `gravity` section of `galaxy.yml`. You will find that the default, if copied from
```

Comment:
This is not setup yet. The [gravity readme](https://github.com/galaxyproject/gravity#configuration) refers to the *galaxy* section of `galaxy.yml` - I assume that's incorrect. In any case, we need to add a `gravity` section to galaxy's config, right?

Reply:
Yeah, there's like 50 todos in here